README.md

```diff
@@ -2,33 +2,33 @@
 
-## Requirements and Setup
+## Prerequisites and Installation
 
-### Requirements
+### Prerequisites
 
-- Docker: The agent is packaged as a Docker image and requires Docker to run.
-- GitHub Enterprise Server: The agent is designed to interact with GitHub Enterprise Server. You need access to a GitHub Enterprise Server instance.
-- GitHub App: The agent operates as a GitHub App. You need to create a GitHub App in your GitHub Enterprise Server instance.
+- **Docker**: The agent is distributed as a Docker image. Docker is required to run it.
+- **GitHub Enterprise Server**: The agent interacts with GitHub Enterprise Server. You need access to a GitHub Enterprise Server instance.
+- **GitHub App**: The agent operates as a GitHub App. You need to create a GitHub App in your GitHub Enterprise Server instance.
 
-### Setup
+### Installation
 
-1. **Create a GitHub App**: Follow the instructions provided by GitHub to create a new GitHub App in your GitHub Enterprise Server instance.
+1. **Create a GitHub App**: Create a new GitHub App in your GitHub Enterprise Server instance. For details, refer to the [GitHub documentation](https://docs.github.com/en/apps/creating-github-apps/about-creating-github-apps/about-creating-github-apps).
 
-2. **Generate a Private Key**: In your GitHub App settings, generate a private key. This will download a PEM file, which the agent will use to authenticate to the GitHub Enterprise environment.
+2. **Generate a private key**: In your GitHub App settings, generate a private key. For instructions, refer to the [GitHub documentation](https://docs.github.com/en/enterprise-cloud@latest/apps/creating-github-apps/authenticating-with-a-github-app/managing-private-keys-for-github-apps). The agent will use the downloaded PEM file to authenticate with the GitHub Enterprise environment.
 
-3. **Install the App**: Install the app on all organizations you want the agent to have access to.
+3. **Install the GitHub App**: Install the app on all organizations that the agent should access. For instructions, refer to the [GitHub documentation](https://docs.github.com/en/apps/using-github-apps/installing-your-own-github-app).
 
-4. **Configure the Agent**: The agent requires several environment variables to run. These could be passed to the Docker command when starting the agent. The required variables are:
+4. **Configure the agent**: The agent requires the following environment variables to run. Pass them to the Docker command when starting the agent.
 
     - `GITHUB_ENTERPRISE_BASE_URL`: The base URL of your GitHub Enterprise Server instance.
     - `GITHUB_APP_ID`: The ID of your GitHub App.
     - `PEM_FILE`: The path to your GitHub App's PEM file inside the Docker container.
 
-5. **Start the Agent**: Run the Docker command to start the agent. Replace `<variable>` with your actual values:
+5. **Start the agent**: To start the agent, run the following Docker command. Replace the variables in angle brackets with your actual values.
```

Comment:
Some suggestions in regards to the Docker image:

- Promote [good use of tags](https://docs.docker.com/reference/cli/docker/image/tag), prefer `leanix/github-agent` from `leanix-github-agent` 
- Promote [version best practices](https://container-registry.com/posts/container-image-versioning/), `docker run ... leanix-github-agent:1.0` is better than `docker run....leanix/github-agent:latest` using latest is [bad](https://vsupalov.com/docker-latest-tag/) and I have seen things going south using latest.